.github/workflows/helm-charts.yaml

```
@@ -200,7 +200,9 @@ jobs:
           --namespace default
           --config etc/ct.yaml
           --charts ${{ matrix.chart_path }}
-          --helm-extra-set-args "--set=kafka.provisioning.useHelmHooks=false"
+          --helm-extra-set-args "
+            --set=kafka.provisioning.useHelmHooks=false \
+            --set=global.deploymentId=testGithub"
```

This only worked by accident (as the `\` is not even needed and forgotten on the first line). As `>-` is used all new lines are replaced by a space, but only if the indentation is the same.  Now it worked as there were quotes, but better to clean it up a little (at least to the extend that yaml allows):

```suggestion
          --helm-extra-set-args "
          --set=kafka.provisioning.useHelmHooks=false
          --set=global.deploymentId=testGithub
          "
```